scalr/data_source_scalr_provider_configuration.go

```diff
@@ -48,6 +49,12 @@ func dataSourceScalrProviderConfiguration() *schema.Resource {
 				Computed:    true,
 				Elem:        &schema.Schema{Type: schema.TypeString},
 			},
+			"owners": {
+				Description: "The teams, the provider configuration belongs to.",
+				Type:        schema.TypeList,
```

Comment:
We have `Set` in the resource because the order doesn't matter and we do not want to update the resource if API returns another order. In the case of data source we can use both and this code is a copy-paste from other similar data source.